js/constants/RepcoinConstants.js

```diff
@@ -3,7 +3,16 @@ var keyMirror = require('keymirror');
 module.exports = {
   ActionTypes: keyMirror({
     GET_CATEGORIES: null,
-    RECEIVE_CATEGORIES: null
+    RECEIVE_CATEGORIES: null,
+    GET_CURRENT_USER: null,
+    RECEIVE_CURRENT_USER: null,
+    GET_LOGGED_IN: null,
+    RECEIVE_LOGGED_IN: null,
```

Comment:
keymirror returns an object with the nulls equal to the string versions of those variable names.